apps/storefront/src/components/B3Tip.tsx

```diff
@@ -42,6 +42,69 @@ function MessageAlert({
   );
 }
 
+const CatalystColors = {
+  success: 'color-mix(in oklab, oklch(83.77% 0.214 142.31), white 75%)',
+  warning: 'color-mix(in oklab, oklch(83.42% 0.159 79.51), white 75%)',
+  info: 'color-mix(in oklab, oklch(49.07% 0.177 262.04), white 75%)',
+  error: 'color-mix(in oklab, oklch(64.89% 0.237 26.97), white 75%)',
+};
+
+function CatalystAlert({ msg, onClose }: AlertProps) {
+  const Body = msg.jsx
+    ? msg.jsx
+    : () => (
+        <span
+          style={{
+            textWrap: 'wrap',
+          }}
+        >
+          {msg.msg}
+        </span>
+      );
+
+  return (
+    <Alert
+      sx={{
+        background: CatalystColors[msg.type],
+        boxShadow: '0 0 #0000, 0 0 #0000, 0 0 #0000, 0 0 #0000, 0 1px 2px 0 rgba(0, 0, 0, 0.05);',
+        borderStyle: 'solid',
+        borderWidth: '1px',
+        borderColor: 'oklab(0.1822 0 0 / 0.1)',
+        borderRadius: '12px',
+        alignItems: 'center',
+        minWidth: '284px',
+        mb: '5px',
+        padding: '12px 12px 12px 16px',
+        color: '#000',
+        fontWeight: 400,
+        letterSpacing: 'normal',
+        fontSize: '14px',
+        fontFamily:
+          'ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji',
+        textWrap: 'auto',
+
+        '& button[title="Close"]': {
+          display: msg.isClose ? 'flex' : 'none',
+        },
+
+        '& .MuiAlert-message': {
```

Comment:
coupling ourselves to an implementation detail of material here

does Material consider its classnames part of an API that it promises not to break between releases?